### ISSUE TITLE
not all tables are confined to article's width

### ISSUE THREAD

**Issue by 2bndy5** [jbms/sphinx-immaterial#141]
I often use `csv-table` or `list-table` for creating tables, but I've noticed that not all tables are confined to the article section's width. When the window is narrow enough for the local Toc to be shown, sometimes I see a table bleed over:
![image](https://user-images.githubusercontent.com/14963867/184448737-c5a74052-ff69-4e17-82b1-cd9ee99d9bce.png)

Adding the CSS,
```css
.md-typeset table.data:not(.plain) {
  /* formerly `display: table;` */
  display: inline-block;
}
```
fixes this issue:
![image](https://user-images.githubusercontent.com/14963867/184449072-31e1033b-cbec-43bf-89e7-736146300b5d.png)

**Comment by jbms** [comment]
Might be interesting to figure out if this behavior is different compared to mkdocs-material

**Comment by 2bndy5** [comment]
I got the idea for `inline-block` by looking at the mkdoc-material's output for tables.

**Comment by jbms** [comment]
Ah yes --- I remember modifying that in _typeset.scss to fix something.  That was before we started adding `/* sphinx-immaterial: */` comments to indicate the reason for changes...

I think the reason for the change was either something to do with wrapping, or something to do with left/right/center alignment of the table itself within the page.

**Comment by 2bndy5** [comment]
fixed in v0.9.0
